testing.rs

```diff
@@ -138,9 +149,22 @@ macro_rules! tests {
                 let start = recorded_events().count();
 
                 assert!(token.transfer(acc.bob, value, vec![]).is_ok());
-                let events = get_events(start);
+                let events = decode_events(start);
+                assert_eq!(events.len(), 1);
+                assert_transfer(&events[0], Some(acc.alice), Some(acc.bob), value);
+            }
+
+            #[ink::test]
+            fn constructor_emits_event() {
```

Comment:
For completeness we should test that if initial supply is 0 we don't emit the event.

Reply:
Added